prow/jobs/kyma/common/common.yaml

```diff
@@ -1,4 +1,4 @@
-# Code generated by rendertemplates. DO NOT EDIT.
+# FIXME(@Ressetkk): adopt this yaml for templates again.
```

Comment:
Is there a ticket for this already or is this a left-over that should not be in this PR anymore when merged?

Reply:
I've added this line because this file was previously generated from a template. Right now due to removed template it's no longer generated. The next step is to adopt this and the other files to the templates.